package.json

```diff
@@ -49,7 +58,9 @@
     "collectCoverageFrom": [
       "src/**/*.{ts,tsx}",
       "!src/index.tsx",
-      "!src/react-app-env.d.ts"
+      "!src/react-app-env.d.ts",
+      "!src/InteractiveMars/Markers.tsx",
+      "!src/InteractiveMars/Mars.js"
```

Comment:
why don't we want test coverage for these?

Reply:
Added those a while ago when the tests were playing up, still works without them though! Will remove in next push